test/integrations/destinations/webhook/processor/data.ts

```diff
@@ -1,3 +1,251 @@
+const geoLocationData = [
+  {
+    name: 'webhook',
+    description: '[GeoLocation] Test 0',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: {
+              anonymousId: 'c82cbdff-e5be-4009-ac78-cdeea09ab4b1',
+              context: {
+                device: {
+                  id: 'df16bffa-5c3d-4fbb-9bce-3bab098129a7R',
+                  manufacturer: 'Xiaomi',
+                  model: 'Redmi 6',
+                  name: 'xiaomi',
+                },
+                network: { carrier: 'Banglalink' },
+                os: { name: 'android', version: '8.1.0' },
+                traits: {
+                  address: { city: 'Dhaka', country: 'Bangladesh' },
+                  anonymousId: 'c82cbdff-e5be-4009-ac78-cdeea09ab4b1',
+                },
+                geo: {
+                  city: 'Gurugram',
+                  country: 'IN',
+                  ip: '223.190.82.63',
+                  location: '28.459700,77.028200',
+                  postal: '122001',
+                  region: 'Haryana',
+                  timezone: 'Asia/Kolkata',
+                },
+              },
+              event: 'spin_result',
+              integrations: { All: true },
+              message_id: 'a80f82be-9bdc-4a9f-b2a5-15621ee41df8',
+              properties: {
```

Comment:
For readability of test case, maintain just 2 properties in the object

Reply:
Done